internal/cmd/run.go

```diff
@@ -317,3 +344,60 @@ func confirmExecution(cmd *cobra.Command, numTasks int, parallel bool, category
 
 	return nil
 }
+
+func promptForEnvVar(cmd *cobra.Command, ev CommandExportExtractMatch) (string, error) {
+	label := fmt.Sprintf("Set Environment Variable \" %s\"", ev.Key)
```

Comment:
```suggestion
	label := fmt.Sprintf("Set Environment Variable \" %s\":", ev.Key)
```

I think this looks cleaner w/ a colon